python/LHCHCGModels.py

```diff
@@ -327,7 +327,8 @@ def setPhysicsOptions(self,physOptions):
         print "BR uncertainties in partial widths: %s " % self.doBRU
     def doParametersOfInterest(self):
         """Create POI out of signal strength and MH"""
-        self.modelBuilder.doVar("kappa_W[1,0.0,2.0]") 
+        self.modelBuilder.doVar("kappa_W[1,0.0,2.0]")
+        kappa_Z = 'kappa_Z'
```

Comment:
Guess this line is hangover from trying to put the total width in as a POI. Can be removed I think.